src/mappings/omim.sssom.tsv

```diff
@@ -217,7 +217,7 @@ MONDO:0007060		skos:exactMatch	OMIM:102530	spermatogenic failure 6	semapv:Unspec
 MONDO:0007061		skos:exactMatch	OMIM:102590		semapv:UnspecifiedMatching
 MONDO:0007062		skos:exactMatch	OMIM:102650	adactylia, unilateral	semapv:UnspecifiedMatching
 MONDO:0007064		skos:exactMatch	OMIM:102700	severe combined immunodeficiency, autosomal recessive, t cell-negative, B cell-negative, nk cell-negative, due to adenosine deaminase deficiency	semapv:UnspecifiedMatching
-MONDO:0007066		skos:exactMatch	OMIM:102800	adenosine triphosphatase deficiency, anemia due to	semapv:UnspecifiedMatching
+MONDO:0007066		skos:exactMatch	OMIM:102800		semapv:UnspecifiedMatching
```

Comment:
Unrelated to this PR, however this OMIM entry, without a label, has been moved to a new entry, https://www.omim.org/entry/102800, so this should not show up as an exactMatch.